.drone.yml

```diff
@@ -246,34 +246,34 @@ pipeline:
       event: tag
       status: success
 
-  publish-vic-machine-server-dev:
-    image: plugins/gcr
+  vic-machine-server-tags:
+    image: '${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.48}'
     pull: true
-    repo: eminent-nation-87317/vic-machine-server
-    dockerfile: cmd/vic-machine-server/Dockerfile
-    custom_dns: 10.142.7.21, 10.142.7.22, 10.166.17.90
-    secrets: [ token ]
-    tags:
-      - dev
+    environment:
+      BIN: bin
+      SHELL: /bin/bash
+    commands:
+      - 'echo -n "$(git describe --tags --abbrev=0)-${DRONE_BUILD_NUMBER}-$(git rev-parse --short=8 HEAD)" > .tags;'
+      - 'if [ "${DRONE_BUILD_EVENT}" = "push" -a "${DRONE_BRANCH}" = "master" ];     then echo ",dev" >> .tags; fi'
+      - 'if [ "${DRONE_BUILD_EVENT}" = "tag"  -a -n "$(echo "${DRONE_BRANCH}" | grep "releases/")" ]; then echo ",$(git describe --tags --abbrev=0),latest" >> .tags; fi'
```

Comment:
I'm honestly not sure, but: would it be better to rely on `grep`'s RC (0 if match, 1 if no match) than using `-n`?

I _think_ you could just have `-a $(echo "${DRONE_BRANCH}" | grep "releases/")`.

Reply:
Hey, yeah, I tried this and the comparison with return codes was not working. Tried with `-eq` (which I didn't expect to work) and with what you have. No 🎲. Using backticks seemed to break the shell too. Some things seem to be really off about the inline shell evaluation.

If I really wanted it to be nice, I would've put everything in a bash script anyways. But it was more important to me to keep the branch/tag logic visible in the drone config.